github-actions/push-to-another-repository/README.md

```diff
@@ -0,0 +1,19 @@
+# Push to another repository
+
+## The problem
+Sometimes we need push code from `Repository A` 
+action to `Repository B` codebase, for eg. when we are implementing
```

Comment:
```suggestion
action to the `Repository B` codebase, for e.g., when we are implementing
```

Reply:
it should be `e.g.` or `for example`, "for" is already included in e.g. :D